tdp_vars_overrides/all/all.yml

```diff
@@ -0,0 +1,2 @@
+---
+ranger_admin_password: RangerAdmin123
```

Comment:
Haha I know but this variable need to be in `all.yml` because all ranger plugin need it for `init` (the POST to Ranger Admin). We need to find a way to remove it from `all.yml`.

Reply:
Right! Good catch, I was wondering why it was here in the first place

Reply:
Open an issue for it https://github.com/TOSIT-IO/tdp-collection/issues/486.